ruby-on-rails/ruby-on-rails-quiz.md

```diff
@@ -969,12 +978,37 @@ end
 
 [Reference](https://guides.rubyonrails.org/layouts_and_rendering.html)
 
-#### Q71. Which choice would you not consider when selecting a gem for your project?
```

Comment:
This question seems too opinionated, therefore I replaced it with a new question.